nidirect_common/src/InvalidateTaxonomyListCacheTags.php

```diff
@@ -50,4 +72,20 @@ public function invalidateForEntity(EntityInterface $entity) {
     }
   }
 
+  /**
+   * Get parent taxonomy term for landing pages.
+   */
+  private function checkLandingPageParent($entity, $tid) {
```

Comment:
Type hints on the parameters recommended. I'd suggest `EntityInterface` and `int` respectively.